apps/nbs-gateway/src/main/java/gov/cdc/nbs/security/oidc/OIDCAuthenticationConfiguration.java

```diff
@@ -21,37 +22,40 @@ SecurityWebFilterChain oidcAuthentication(
     return http
         .authorizeExchange(
             authorize ->
-            //  the landing page
-            authorize.pathMatchers(
-                HttpMethod.GET,
-                "/welcome/**").permitAll()
-                //  paths associated with authentication
-                .pathMatchers(
-                    HttpMethod.GET,
-                    "/nbs/logged-out",
-                    "/nbs/timeout",
-                    "/nbs/logOut")
-                .permitAll()
-                //  assets that do not require authentication
-                .pathMatchers(
-                    HttpMethod.GET,
-                    "/nbs/*.js",
-                    "/nbs/*.css",
-                    "/nbs/*.gif",
-                    "/nbs/task_button/**",
-                    "/images/nedssLogo.jpg",
-                    "/favicon.ico",
-                    "/static/**",
-                    "/logout",
-                    "/goodbye",
-                    "/expired"
-                )
-                .permitAll()
-                .anyExchange()
-                .authenticated())
+                //  the landing page
```

Comment:
looks like no change here... we definitely need to figure out how to sync our editor/indentations :-)

Reply:
The `/` path was added in there to prevent the root from requiring authorization.  There must have been tab changes also to catch that whole block.  Just switch to Intellij 😄